src/mmw/mmw/settings/bigcz_settings.py

```diff
@@ -0,0 +1 @@
+BIGCZ_CLIENT_TIMEOUT = 5 # timeout in seconds
```

Comment:
Instead of creating a new file, prefer putting this near [the other BIGCZ configuration in `base.py`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/mmw/settings/base.py#L360)